### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Phyloiota**
changed solutions to "frameworks and uploaded images from IOTA slides

# Description of change

Please write a summary of your changes and why you made them. 

## Links to any relevant issues

Be sure to reference any related issues by adding `fixes issue #`.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Bug fix (a non-breaking change which fixes an issue)
- Enhancement (a non-breaking change which adds functionality)
- Documentation Fix
- Documentation Enhancement

## How the change has been tested

Describe the tests that you ran to verify your changes.

Make sure to provide instructions for the maintainer as well as any relevant configurations.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have tested the wiki locally and tested that all external links work

### CODE REVIEW

File: docs/learn/about-iota/an-introduction-to-iota.md [hunk 1]
```diff
@@ -20,6 +20,10 @@ The main differences between blockchains and the Tangle are the following:
 
 To explain these points, you need to understand the differences between the data structures and the consensus mechanisms in these DLTs.
 
+**Blockchain vs. Tangle**
+
+![layer1-tangle](/mg/learn/layer1-tangle.png)
```
Comment (Dr-Electron) [inline docs/learn/about-iota/an-introduction-to-iota.md:25]:
```suggestion
![layer1-tangle](/img/learn/layer1-tangle.png)
```

File: docs/participate/frameworks/introduction.md [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
 
-The IOTA foundation maintains and develops a collection of solutions building on the Tangle.
+The IOTA foundation maintains and develops a collection of frameworks building on the Tangle.
 
-## List of Solutions
+## List of Frameworks
+
+![Layer2-frameworks](/img/learn/layer2-frameworks.png)
 
 - [Streams](/docs/participate/solutions/streams):
```
Comment (Dr-Electron) [inline docs/participate/frameworks/introduction.md:14]:
```suggestion
- [Streams](/docs/participate/frameworks/streams):  
```

Comment (Dr-Electron) [reply docs/participate/frameworks/introduction.md:14]: Same for the other links in this page please :)

**Review by Dr-Electron — APPROVED** [review]
Nice graphics ❤️

**Review by Phyloiota — COMMENTED** [review]
all fine - thanks

**Pull request merged**